content/en/security/application_security/api-inventory/_index.md

```diff
@@ -45,49 +65,67 @@ To use API Security on your services, **you must have AAP Threats Protection ena
 
 **Note**: On .NET Core and .NET Fx tracers, you need to set the environment variable `DD_API_SECURITY_ENABLED=true` for API Security features to work properly.
 
-## How it works
-
-API Inventory leverages the Datadog tracing library with AAP enabled to gather security metadata about API traffic, including the API schema, types of sensitive data processed, and the authentication scheme. API information is evaluated per endpoint, every 30 seconds, which should ensure minimal performance impact.
+API Endpoints leverages the Datadog tracing library with App and API Protection enabled to gather security metadata about API traffic, including the API schema, types of sensitive data processed, and the authentication scheme. API information is evaluated per endpoint, every 30 seconds, which should ensure minimal performance impact.
 
-API Inventory Security uses [Remote Configuration][1] to manage and configure scanning rules that detect sensitive data and authentication.
+API Endpoints uses [Remote Configuration][1] to manage and configure scanning rules that detect sensitive data and authentication.
 
 The following risks are calculated for each endpoint:
 
-### Security trace activity
+### Data sources
+
+In the **API Endpoints** explorer, the **Data Sources** show where visibility originates.
+
+The following data sources are explored.
+
+#### AWS API Gateway
+
+The **AWS API Gateway** data source provides a per-endpoint risk view that confirms exposure, traces the source, and remediates at the service or API Gateway level.
```

Comment:
The API Gateway source works slightly in a similar manner to the API Definition in that a customer has to define their configuration within AWS. This configuration is then consumed by one of our services, so it is technically a definition. As of now there is little correlation with traffic, although we're working on it.